src/components/Navbar/NavItem.tsx

```diff
@@ -0,0 +1,14 @@
+import Link from 'next/link';
+
+type NavItemsProps = {
+  navItemSrc: string;
+  navItem: string;
+};
+
+const NavItem = ({ navItemSrc, navItem }: NavItemsProps) => (
+  <Link href={navItemSrc}>
+    <li className="whitespace-nowrap p-2 text-[16px]">{navItem}</li>
+  </Link>
```

Comment:
I wonder if we need `text-[16px]` :)

Please add a proper font-color from the designs, from what I see - for most of the elements it's `#333333` if you want you may verify that and add it as global, for nav items it's `#636B75`